logging/google/cloud/logging/logger.py

```diff
@@ -436,29 +462,34 @@ def log_proto(self, message, labels=None, insert_id=None, severity=None,
         """
         self.entries.append(
             ('proto', message, labels, insert_id, severity, http_request,
-             timestamp))
+             timestamp, resource))
 
-    def commit(self, client=None):
+    def commit(self, client=None, resource=None):
```

Comment:
I'd say let's leave it off for now and just have users specify it on the entry level. If there were any other properties that we did this way, I'd be for it, but I'd say let's leave it out until someone asks for it. :)